apps/earn-protocol/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@summerfi/earn-protocol",
+  "name": "@summerfi/earn-protocol-webapp",
```

Comment:
This change will break the earn build, we need to update `--filter=@summerfi/earn-protocol` in main package.json

Reply:
Updated, good catch 🪝 thanks